parteh/PRTLossFluxesMod.F90

```diff
@@ -885,7 +886,62 @@ subroutine PRTDamageRecoveryFluxes(prt, organ_id, mass_0, mass, cc_mass)
          + (mass - mass_0)
 
     end subroutine PRTDamageRecoveryFluxes
-      
+
+     ! =====================================================================================
+
+  subroutine PRTHerbivoryLosses(prt, organ_id, mass_fraction)
+
+    ! ----------------------------------------------------------------------------------
+    ! This subroutine assumes that there is no re-translocation associated
+    ! with herbivory. There is only one destiny for eaten mass within
+    ! the organ, and that is outside the plant.  
+    ! It is also assumed that non PARTEH parts of the code (ie the grazing-model)
+    ! will decide what to do with the consumed mass (i.e. sent it to the litter
```

Comment:
Just for clarity and ease of understanding, maybe explicitly say where the “grazing-model” is for tracking and clarity. Is this “subroutine FatesGrazing” you mean or something else?